src/breadcrumbs/styled-components.js

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import {styled} from '../styles/index';
+import type {StyledRootPropsT, StyledSeparatorT} from './types';
+import {ChevronRight} from '../icon';
+
+export const StyledRoot = styled('nav', ({$theme}: StyledRootPropsT) => {
+  return {
+    color: $theme.colors.mono900,
+    ...$theme.typography.font350,
+  };
+});
+
+export const StyledSeparator = styled('div', ({$theme}: StyledSeparatorT) => {
+  return {
+    display: 'inline-block',
+    color: $theme.colors.mono700,
```

Comment:
can can add a breadcrumb specific color, so it would work with both the light and dark theme? something like this: https://github.com/uber-web/baseui/blob/834307495cc9a3803acc09735e693ebef52811ee/src/themes/creator.js#L173

Reply:
I've added the `breadcrumbsText` and `breadcrumbsSeparatorFill` to the theme.

Here is an updated dark mode:

<img width="391" alt="screen shot 2018-11-29 at 1 21 28 pm" src="https://user-images.githubusercontent.com/4030377/49252807-d9a93d00-f3d9-11e8-9120-7a6cd27df0d3.png">